configs/dovecot/conf.d/10-master.conf.sample

```diff
@@ -54,17 +48,12 @@ service imap {
   #vsz_limit = $default_vsz_limit
 
   # Max. number of IMAP processes (connections)
-  #process_limit = 1024
+  process_limit = 0
```

Comment:
Document, what ``process_limit = 0`` actually does.

From the top of my head: It uses some default value of Dovecot.